### ISSUE TITLE
[Citations] Have a way to track all citations and mark those that are unused

### ISSUE THREAD

**Issue by Fleker** [Fleker/rafflesia#1]
Like a linter, add warnings for unused citations.

**Comment by anselmobattisti** [comment]
Your ideia here is to identify the entries in the .bib file that are not mentioned in the .tex files?

**Comment by Fleker** [comment]
Yeah if this project could temporarily store all the citations, or at least the keys from `.bib` files, then it could look through `.tex` files and check which keys are being used.
